audformat/core/utils.py

```diff
@@ -286,6 +286,95 @@ def intersect(
     return index
 
 
+def join_labels(
+        labels: typing.Sequence[typing.Union[typing.List, typing.Dict]],
+):
+    r"""Combine scheme labels.
+
+    This might be helpful,
+    if you would like to combine two databases
+    that have the same scheme,
+    but with different labels:
+
+    .. code-block:: python
+
+        labels = audformat.utils.join_labels(
+            [
+                db.schemes['scheme'].labels,
+                db_new.schemes['scheme'].labels,
+            ]
+        )
+        db.schemes['scheme'].replace_labels(labels)
+        db_new.schemes['scheme'].replace_labels(labels)
+        db.update(db_new)
+
+    Args:
+        labels: sequence of labels to join
+
+    Returns:
+        joined labels
+
+    Raises:
+        ValueError: if labels argument is a dict
+        ValueError: if labels are of different type
+        ValueError: if the labels are dicts
+            and contain different values for the same key
+        RuntimeError: if label type is not ``list`` or ``dict``
+
+    Example:
+        >>> join_labels([{'a': 0, 'b': 1}, {'b': 1, 'c': 2}])
+        {'a': 0, 'b': 1, 'c': 2}
+
+    """
+    if isinstance(labels, dict):
```

Comment:
Why do you think we need this check? The docstring clearly states that the user has to pass a list.

Reply:
No, you are right, I removed it.